kuebiko/pipeline/companies_house/utils.py

```diff
@@ -0,0 +1,92 @@
+"""Companies House data processing."""
+import pandas as pd
+
+
+COLUMN_MAPPINGS = {
+    "CompanyName": "company_name",
+    " CompanyNumber": "company_number",
+    "URI": "uri",
+    "RegAddress.CareOf": "address_careof",
+    "RegAddress.POBox": "address_pobox",
+    "RegAddress.AddressLine1": "address_line1",
+    " RegAddress.AddressLine2": "address_line2",
+    "RegAddress.PostCode": "address_postcode",
+    "RegAddress.PostTown": "address_town",
+    "RegAddress.County": "address_county",
+    "RegAddress.Country": "address_country",
+    "CompanyCategory": "company_category",
+    "CompanyStatus": "company_status",
+    "CountryOfOrigin": "country_of_origin",
+    "DissolutionDate": "dissolution_date",
+    "IncorporationDate": "incorporation_date",
+    "SICCode.SicText_1": "sic_1",
+    "SICCode.SicText_2": "sic_2",
+    "SICCode.SicText_3": "sic_3",
+    "SICCode.SicText_4": "sic_4",
+}
+
+
+def read_from_url(url: str) -> pd.DataFrame:
+    """Read Companies House zipped CSV data-dump from url."""
+    return (
+        pd.read_csv(url, usecols=COLUMN_MAPPINGS.keys(), compression="zip")
```

Comment:
I noticed that elsewhere you've used `zipfile.open` to open the compressed table rather than pandas `compression` variable (ie, in `pipeline/nspl/utils.py`). Are there reasons for using one or the other approach in different cases?